docs/deep-dive/02-dd-create-git-function-serverless.md

```diff
@@ -98,12 +98,12 @@ Follows these steps:
       type: git
       runtime: nodejs12
       source: $GIT_FUNCTION
-      reference: master
+      reference: main
       baseDir: orders-service/function
     EOF
     ```
 
-    >**NOTE:** See this [Function's code and dependencies](https://github.com/kyma-project/examples/tree/master/orders-service/function).
+    >**NOTE:** See this [Function's code and dependencies](https://github.com/kyma-project/examples/tree/main/orders-service/function).
 
 5. Check if your Function was created and all conditions are set to `True`:
```

Comment:
`You should get a result similar to this example:` -> change also below if you decide to change it in other places